src/Server.h

```diff
@@ -144,6 +152,9 @@ class cServer
 	from the settings. */
 	bool ShouldAllowMultiWorldTabCompletion(void) const { return m_ShouldAllowMultiWorldTabCompletion; }
 
+	/** Get the Forge mods registered for a given protocol. */
+	AStringMap & GetRegisteredForgeMods(const UInt32 a_Protocol);
```

Comment:
I tried changing to a public const accessor with a private non-const function to get the mods for modification, but my 2¢ it seems less elegant: https://github.com/satoshinm/cuberite/pull/2/commits/aa596db7fa34abb66852d4f3fe0e5b845113f2fc so I reverted it, @madmaxoft @lkolbly let me know what you think